README.md

```diff
@@ -2,18 +2,15 @@
 
 This repo contains python source code for converting data files in the [BIDS-NIRS-Tapping](https://github.com/rob-luke/BIDS-NIRS-Tapping) dataset from [SNIRF](https://github.com/fNIRS/snirf) format to [NWB](https://www.nwb.org/nwb-neurophysiology/) format using the [ndx-nirs](https://github.com/agencyenterprise/ndx-nirs) extension. In addition, scripts for arranging the files and uploading to the [DANDI archive](https://gui.dandiarchive.org/) are included.
 
-**Note:** this repo is at a very early WIP stage. Proper organization, documentation, unit tests, etc will be added with time.
+**Note:** this repo is at a WIP stage. Improved organization, documentation, unit tests, etc are expected to be added with time.
 
 
-## Installation
-
-Note: eventually this will be converted over to install via setup.py or poetry
+## Install Dependencies
 
 ```bash
 $ pip install -r requirements.txt
 ```
 
-
 ## Get Dataset
 
 ```bash
```

Comment:
This command will work just fine but will default to the latest version of the dataset on git, right? Would it make more sense to have this clone a specific commit or use a wget command + unzip, something like
```bash
$ wget https://github.com/rob-luke/BIDS-NIRS-Tapping/archive/388d2cdc3ae831fc767e06d9b77298e9c5cd307b.zip data/BIDS-NIRS-Tapping.zip
$ unzip data/BIDS-NIRS-Tapping.zip
```
Below in the documentation, you specify this commit, so it might be worth the consistency.

Reply:
good idea! Thanks!